src/i18n/index.ts

```diff
@@ -0,0 +1,52 @@
+import i18next from 'i18next';
+import {list} from '../plugin';
+import {EP_PHOVEA_CORE_LOCALE, ILocaleEPDesc} from '../extensions';
+
+const DEFAULT_LANGUAGE = 'en';
+const DEFAULT_NAMESPACE = 'default_namespace';
```

Comment:
Why is the value `default_namespace`?

Reply:
The value  `defaut_namespace`  is entirely random , it could be anything .
The point of it is that  if u don't want to  define a namespace so you can use the syntax 
`i18n.t(key)` instead of `i18n.t(namespace:key)`, you can .